siphon/catalog.py

```diff
@@ -393,7 +399,9 @@ def latest(self):
                 return TDSCatalog(latest_cat).datasets[0]
         raise AttributeError('"latest" not available for this catalog')
 
-    __repr__ = __str__
+    def __repr__(self):
+        """Return informative repr for TDSCatalog."""
+        return str(TDSCatalog) + ' ' + self.catalog_name
```

Comment:
Isn’t this just returning then `self.catalog_name + ‘ ‘ + self.catalog_name`?

Reply:
Updated this to a new return